.env.test

```diff
@@ -5,6 +5,9 @@ INFLUX_PORT=8086
 INFLUX_TOKEN_WRITE=my-token
 INFLUX_ORG=my-org
 INFLUX_BUCKET=my-bucket
+INFLUX_MEASUREMENT=my-measurement
```

Comment:
Sollten wir in der Doku beschreiben.

SENEC
SUNROW

Reply:
Wie das Measurement heißt, ist prinzipiell egal, es muss nur überall korrekt eingestellt sein. Das heißt, dass Importer und Collector und Dashboard auf das gleiche Measurement zugreifen müssen. 

Ursprünglich war das Measurement für die PV-Messwerte statisch und lautete "SENEC". Seit v0.11.0 von SOLECTRUS kann es angepasst werden:
https://github.com/solectrus/solectrus/issues/2031

Wer mit einem bestimmten Measurement gestartet ist und Daten gesammelt hat, muss leider dabei bleiben - InfluxDB bietet kaum eine Möglichkeit, ein Measurement in den Daten umzubenennen (außer manuelles Umkopieren).

In der Anleitung habe gerade einen Hinweis ergänzt:
https://github.com/solectrus/hosting/commit/62972cd3fed95269ad4db445b02564895063f80e

Reply:
Passt. Merg und Release :-)